CMakeLists.txt

```diff
@@ -92,7 +92,9 @@ if(NOT DEFINED EIGEN3_INCLUDE_DIR)
   find_package(Eigen3 REQUIRED)
 endif()
 target_include_directories(lcontract SYSTEM PUBLIC ${EIGEN3_INCLUDE_DIR})
-target_compile_definitions(lcontract PUBLIC EIGEN_DONT_PARALLELIZE)
+## in order to combine threading over time slices for eigenvector I/O 
+## and threading in Eigen for dense matrix algebra, we need to enable this
+#target_compile_definitions(lcontract PUBLIC EIGEN_DONT_PARALLELIZE)
```

Comment:
It might be that this requires some more subtlety: for the contractions, we probably want to have ```EIGEN_DONT_PARALLELIZE``` set (since we can't use nested parallelism anyway as far as we can tell right now), while for ```VdaggerV``` we of course want to have it set.

Can you benchmark how the contractions behave when it is NOT set and you run with 8 threads on lnode07, for example?

Reply:
It would be a shame if we gain some factor in the separate VdaggerV computation but lose substantially in the contractions as a result.

Reply:
On lnode07 I can just run the code, or have to set a jobscript for that?

Reply:
Did you guys not speak to each other? The node is in ```drain``` state and can be used for testing directly.